src/components/Movie/Movie.spec.js

```diff
@@ -0,0 +1,14 @@
+import {render, screen} from '@testing-library/react';
+
+import Movie from './Movie';
+
+describe('Movie', () => {
+  it('Shows that the movie poster is being renderd', () => {
+    render(<Movie />);
+    screen.getByRole('img');
+  });
+  it('Shows that the movie title is being renderd', () => {
+    render(<Movie />);
+    screen.getByRole('heading');
+  });
+});
```

Comment:
Same as in your previous test - please have a look at my comment in your other test file.